sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -660,12 +660,21 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+
+  val QUERY_EXECUTION_LISTENERS =
+    ConfigBuilder("spark.sql.queryExecutionListeners")
+      .doc("QueryExecutionListeners to be attached to the SparkSession")
```

Comment:
In this case I updated the doc to read "A comma-separated list of classes that implement QueryExecutionListener that will be attached to the SparkSession". I could attach the whole line I put in sql-programming-guide.md but it will make it look out of place compared to the docs for other properties in the same class.